index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css"/>
+  <title>Portfolio</title>
+</head>
+<body>
+  <main class="container">
+  
+    <header class="header">
+      <ul>
+        <li><img src="./assets/Time.png" alt="clock" class="top"></li>
+        <li><img src="./assets/Group 3.png" alt="bars" class="top"></li>
+      </ul>
```

Comment:
- [ ] Great job 👍 . But, this part of the design isn't necessary. It was just added to the Figma design to imitate a user's mobile screen. Kindly remove this part 🙏 

![image](https://user-images.githubusercontent.com/74814780/198427212-f165ce39-f4c6-42fd-be79-e2e2d16614f1.png)